test/jdk/java/util/TimeZone/CustomTzIDCheckDST.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2001, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
This is a new test case, the year should be only 2022.

Reply:
Thanks, corrected this in the following commit